autoPlay.user.js

```diff
@@ -1073,6 +1100,10 @@ function getClickDamage(){
     return g_Minigame.m_CurrentScene.m_rgPlayerTechTree.damage_per_click;
 }
 
+function autoRefreshPage(autoRefreshSeconds){
+	refreshTimer = setTimeout("location.reload(true);",(autoRefreshSeconds*1000));
```

Comment:
Couple of nits here: suggest using `function(){w.location.reload(true);}` instead of an eval'ed string (I'd be surprised if JSHint allowed that), also unnecessary parentheses around the multiplication.